wagtail/admin/templates/wagtailadmin/shared/page_status_tag.html

```diff
@@ -1,14 +1,14 @@
+{% load wagtailcore_tags %}
 {% load i18n wagtailadmin_tags %}
 {% if page.live %}
-    {% with page_url=page.url %}
-        {% trans "Current page status:" as status_hidden_label %}
-        {% if page_url is not None %}
-            {% trans 'Visit the live page' as status_title %}
-            {% status page.status_string url=page_url title=status_title hidden_label=status_hidden_label classname="primary" attrs='target="_blank" rel="noreferrer"' %}
-        {% else %}
-            {% status page.status_string hidden_label=status_hidden_label classname="primary" %}
-        {% endif %}
-    {% endwith %}
+    {% pageurl page as page_url %}
+    {% trans "Current page status:" as status_hidden_label %}
+    {% if page_url is not None %}
+        {% trans 'Visit the live page' as status_title %}
+        {% status page.status_string url=page_url title=status_title hidden_label=status_hidden_label classname="primary" attrs='target="_blank" rel="noreferrer"' %}
+    {% else %}
+        {% status page.status_string hidden_label=status_hidden_label classname="primary" %}
+    {% endif %}
 {% else %}
     {% status page.status_string hidden_label=status_hidden_label %}
```

Comment:
Not an issue with your PR but I noticed that `status_hidden_label` will never be anything in this branch, so I will pull this variable up into the parent scope (above the if statement) in this PR.